src/schema/v2/artist/helpers.ts

```diff
@@ -92,6 +92,15 @@ export const ARTIST_INSIGHT_MAPPING: Record<
     getEntities: (artist) => artist.vanguard_year && [],
     getLabel: (artist) => `The Artsy Vanguard ${artist.vanguard_year}`,
   },
+  GAINING_FOLLOWERS: {
+    getDescription: (artist) =>
+      `${artistNewFollowersPercentage(
+        artist
+      )}% increase in Artsy followers compared to same time last year.`,
+    getEntities: (artist) =>
+      artistNewFollowersPercentage(artist) >= 20 ? [] : null,
```

Comment:
**question [non-blocking]:** 

I'm confused about this line - why would we want to sometimes return an empty array and other times return null? And what does the number 20 signify? 🤔 

*format via [Conventional Comments](https://conventionalcomments.org/)*

Reply:
:ceiling-cat: I don't wanna block any work, but shouldn't all this data come from Gravity? I know sometimes we manipulate somehow some data in MP, but I think it makes sense to make it totally dumb, isn't it?

Reply:
It could be just a virtual field there

Reply:
Right, initially I was thinking it will be useful to calculate the value in MP but look like it isn't necessary and can be moved to Gravity.